model_analyzer/config/generate/neighborhood.py

```diff
@@ -133,14 +185,30 @@ def _clip_vector_values(self, vector: Coordinate,
 
         if max_value > clip_value and max_value != 0:
             for i in range(len(vector)):
-                vector[i] = clip_value * vector[i]/max_value
+                vector[i] = clip_value * vector[i] / max_value
         return vector
 
     def pick_coordinate_to_initialize(self) -> Optional[Coordinate]:
         """
         Based on the initialized coordinate values, pick an unvisited
         coordinate to initialize next.
+
+        If the neighborhood is in slow mode, only pick from within the next door neighbors
```

Comment:
Is "next door" defined anywhere? Would it be better to be explicit and say adjacent neighbors?

Reply:
updated